documentation/DCP-documentation/config_examples.md

```diff
@@ -43,7 +43,7 @@ Our internal configurations for each pipeline are as follows:
 | TASKS_PER_MACHINE | 1 | 1 | 1 | 1 | 1 |  |
 | MACHINE_TYPE | ['c5.xlarge'] | ['c5.xlarge'] | ['c5.xlarge'] | ['c5.xlarge'] | ['c5.xlarge'] | Historically we have used m4.xlarge and then m5.xlarge however very recently we have been having a hard time getting m class machines so we have switched to c class. Note that they have different memory sizes so you need to make sure MEMORY is set correctly if changing between classes. |
 | MACHINE_PRICE | .20 | .20 | .20 | .20 | .20 | Will be different for different size/classes of machines. |
-| EBS_VOL_SIZE | 22 | 22 | 22 | 22 | 22 | You might need to make this larger if you set DOWNLOAD_FILES = True  |
+| EBS_VOL_SIZE <br>(if using mounted volume)<br><br>(if downloading from external bucket) | 22<br><br>22 | 22<br><br>200 | 22<br><br>22 | 22<br><br>22 | 22<br><br>40 | Suggested size increases when downloading files from another bucket (DOWNLOAD_FILES = True) depending on the files. |
```

Comment:
@emiglietta Not sure what text editor you did this in, but it looks like it some weirdness happened! 

Please do just edit the notes section, not the parameter name section - the edit is also a bit misleading, because all instances DO have an EBS volume (it's essentially the hard drive - like a regular computer, all machines must have one, because that's where the operating system is located!) We don't actually add other EBS volumes as "external drives" here, it's just a matter of how big of an internal drive we have.

Reply:
Sure, thanks!